tequilapi/endpoints/sessions.go

```diff
@@ -102,21 +104,28 @@ func AddRoutesForSession(router *httprouter.Router, sessionStorage sessionStorag
 	router.GET("/sessions", sessionsEndpoint.List)
 }
 
-func sessionToDto(se connection.Session) sessionDTO {
-	return sessionDTO{
+func sessionToDto(se connection.Session) SessionDTO {
+	return SessionDTO{
 		SessionID:       string(se.SessionID),
 		ProviderID:      string(se.ProviderID.Address),
 		ServiceType:     se.ServiceType,
 		ProviderCountry: se.ProviderCountry,
 		DateStarted:     se.TimeStarted.Format("2018-10-29 16:22:05"),
-		BytesSent:       se.DataStats.BytesSent,
-		BytesReceived:   se.DataStats.BytesReceived,
-		Duration:        se.Duration,
+		BytesSent:       uint64(se.DataStats.BytesSent),
+		BytesReceived:   uint64(se.DataStats.BytesReceived),
+		Duration:        getDuration(se.TimeStarted, se.TimeUpdated),
```

Comment:
move the getDuration method to connection.Session struct, that way you can just call se.GetDuration()

Reply:
moved